telegram/commands/report/shower.go

```diff
@@ -0,0 +1,132 @@
+package report
+
+import (
+	"activity-tracker/shared"
+	"activity-tracker/storage"
+	"fmt"
+	"time"
+)
+
+var (
+	reportShowerMessage = `Después de ir a echarle un ojo al baño me encontré este resumen precios@ %s 🛀🏻
+
+Reporte semanal:
+Lunes: %s
+Martes: %s
+Miércoles: %s
+Jueves: %s
+Viernes: %s
+Sábado: %s
+Domingo: %s
+
+Última racha:
+Bañándose: %d %s 🚿
+Sin bañarse: %d %s 🤢
+
+Andá bañate pa' que estos números mejoren, cochin@ de 💩💩`
+
+	oneDayLabel        = "día"
+	textSisasStreak    = "días"
```

Comment:
sisas = dias? what the heck?

Reply:
please look at the logic, the name is correct